app/scripts/modules/kubernetes/src/serverGroup/paramsMixin.js

```diff
@@ -8,7 +8,7 @@ module.exports = angular
 
     function destroyServerGroup(serverGroup) {
       return {
-        namespace: serverGroup.namespace,
+        namespace: serverGroup.region,
```

Comment:
This definitely worked at some point under some conditions. Could you please make this 
```
namespace: serverGroup.region || serverGroup.namespace,
```
instead?

Reply:
Yeah, I started to do that at first, but the other 2 are just `serverGroup.region` so I went with this. Should I go ahead and change all 3?

Reply:
Hmm. Yes? I guess that feels best. Thanks.